### DISCUSSION TITLE
Automating Formula Update Process for Liquibase

### DISCUSSION THREAD

**Discussion by jandroav** [Homebrew/discussions#5151] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Hello Homebrew Maintainers,

As a member of the Liquibase DevOps team, we're constantly looking for ways to streamline our release processes and ensure that our software is easily accessible to users across different platforms. Recently, we've expanded our release automation to include Docker, Debian, RPM, and Chocolatey packages directly from our GitHub workflows whenever a new Liquibase release is made.

Now, we're looking to integrate this workflow seamlessly with Homebrew. We want to ensure that Homebrew formula updates are automated efficiently and align with the best practices recommended by the Homebrew community.

We believe that opening a pull request (PR) with a commit similar to the following example would be the most straightforward way to update the formula: https://github.com/Homebrew/homebrew-core/commit/7527545aec089a704352b3b31fc088b1fcb7b1e0

Could you please confirm if this is indeed the recommended approach? If not, we'd appreciate guidance on the best way to automate the formula update process for Liquibase within the Homebrew ecosystem.

Looking forward to your insights and suggestions.

Thanks!

**Comment by jandroav** [comment]
Pinging here @Porkepix and @fxcoudert

**Reply by SMillerDev** [reply]
No need to ping people unless they have expressed specific interest.

**Comment by SMillerDev** [comment, marked as answer]
https://github.com/marketplace/actions/bump-homebrew-formula or https://github.com/marketplace/actions/homebrew-bump-formula are the best way to bump a formula once your project makes a release.

**Reply by jandroav** [reply]
Thanks a lot, @SMillerDev !
